first-uppercase.js

```diff
@@ -0,0 +1,15 @@
+// Napisz funkcję firstUppercase(), która przyjmie zdanie i zamieni pierwszą literę każdego
+// słowa na wielką.
+
+function firstUppercase(string) {
+    const words = string.split(" ");
+
+    for (let i = 0; i < words.length; i++) {
+        words[i] = words[i].charAt(0).toUpperCase() + words[i].substring(1);
```

Comment:
It's ok but it's better to use [template literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals).